test/integration/docker-compose-client.yml

```diff
@@ -33,6 +33,7 @@ services:
       BEYLA_METRICS_INTERVAL: "10ms"
       BEYLA_BPF_BATCH_TIMEOUT: "10ms"
       BEYLA_LOG_LEVEL: "DEBUG"
+      BEYLA_ENFORCE_SYS_CAPS: "false"
```

Comment:
Did you notice any issue with the capabilities enforcement in docker? Will tests fail if we remove this line?

Reply:
No, I just changed this one integration test to ensure that integrations tests are not failing when the flag is `false` (because it's defaulted to `true` and that's what the other integration tests are using)